meta-oe/recipes-devtools/sqlite-orm/sqlite-orm_1.5.bb

```diff
@@ -0,0 +1,22 @@
+SUMMARY = "SQLite ORM light header only library for modern C++"
+HOMEPAGE = "https://github.com/fnc12/sqlite_orm"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=3b5bf5ee8bb16d8e19359efe11bdc3f2"
+
+inherit cmake
+
+DEPENDS += "sqlite3"
+
+SRC_URI = "https://github.com/fnc12/sqlite_orm/archive/v1.5.tar.gz"
```

Comment:
github archives can be re-generated occasionally and change the checksums, therefore in OE we do not prefer archive, releases are ok. So I would suggest to use 

```
SRCREV = "e8a9e9416f421303f4b8970caab26dadf8bae98b"
SRC_URI = "git://github.com/fnc12/sqlite_orm;protocol=https"
S = "${WORKDIR}/git"
```